lib/css_parser/rule_set.rb

```diff
@@ -634,6 +639,21 @@ def parse_selectors!(selectors) # :nodoc:
         s
       end
     end
+
+    def split_value_preserving_function_whitespace(value)
+      split_value = value.gsub(RE_FUNCTIONS) do |c|
+        c.gsub!(/\s+/, WHITESPACE_REPLACEMENT)
+        c
+      end
+
+      matches = split_value.strip.split(/\s+/)
+
+      matches.each do |c|
+        c.gsub!(WHITESPACE_REPLACEMENT, ' ')
+      end
+
+      matches
```

Comment:
each already returns matches
```suggestion
      matches.each do |c|
        c.gsub!(WHITESPACE_REPLACEMENT, ' ')
      end
```